BlueprintUI.podspec

```diff
@@ -19,6 +19,10 @@ Pod::Spec.new do |s|
 
   s.weak_framework = 'SwiftUI'
 
+  s.pod_target_xcconfig = {
+    'APPLICATION_EXTENSION_API_ONLY' => 'YES',
```

Comment:
As far as I can tell, there's no equivalent for SPM. 
It's supposed to inherit this flag from its dependencies.

Reply:
The _closest_ is an unsafe linker flag, as documented [here](https://danielsaidi.com/blog/2022/05/18/how-to-suppress-linking-warning)